README.md

```diff
@@ -1 +1,24 @@
 # Node.js esclient module
+
+[![build:?](https://img.shields.io/travis/Enrise/node-esclient.svg?style=flat-square)](https://travis-ci.org/Enrise/node-esclient)
+[![Coverage Status](https://img.shields.io/coveralls/Enrise/node-esclient/master.svg?style=flat-square)](https://coveralls.io/github/Enrise/node-esclient?branch=master)
+[![dependencies:?](https://img.shields.io/david/Enrise/node-esclient.svg?style=flat-square)](https://david-dm.org/Enrise/node-esclient)
+[![devDependencies:?](https://img.shields.io/david/dev/Enrise/node-esclient.svg?style=flat-square)](https://david-dm.org/Enrise/node-esclient)
+
+> A simple wrapper around the [elasticsearch](https://github.com/elastic/elasticsearch-js) client with integrated [enrise-logger](https://github.com/Enrise/node-logger).
+
+### Installation
+NPM: `npm install enrise-esclient enrise-logger --save`  
```

Comment:
It is only a necessity if you want to configure the logger, right? And I would prefer that situation to be handled via the configuration of the ESClient. I find it awkward to have to have a separate package required and configured for this package to work.